strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.9__Repository_group_repositories.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog xmlns:o="http://www.unbroken-dome.org/schema/liquibase-orientdb"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <changeSet id="v1.0.0.9" author="przemyslaw.fusik@gmail.com">
+
+        <o:dropProperty propertyName="groupRepositories" className="Repository"/>
```

Comment:
Your IDE may show an error on this line, there is a minor bug in the schema (https://github.com/unbroken-dome/liquibase-orientdb/issues/3)